pallets/nft-manager/src/lib.rs

```diff
@@ -774,37 +771,6 @@ pub mod pallet {
 
             Ok(())
         }
-
-        #[pallet::weight(T::DbWeight::get().reads_writes(1 as u64, 2 as u64))]
```

Comment:
TODO:
Ensure this is completely safe and that the call_index can be reused. If not, find out how to "burn it"

Reply:
Based on [this discussion on stackexchange](https://substrate.stackexchange.com/questions/8923/safe-way-to-remove-an-extrinsic):
We can remove the extrinsic and reuse the index provided we give enough time to clients to upgrade. It is safe though in the blockchain context.